### ISSUE TITLE
How to use the prerender-spa-plugin to remove the slash when you refresh the page?

### ISSUE THREAD

**Issue by Sergey-Ssn** [chrisvfritz/prerender-spa-plugin#307]
demo http://appvue.pickstyle.ru if /page refresh add slash and when you refresh the page, the page breaks down. If Apache does this, what settings should be added to the file .htaccess?

**Comment by 1823616178** [comment]
Have you solved it? I have the same problem

**Comment by Sergey-Ssn** [comment]
> Have you solved it? I have the same problem

Yes, a simple solution found in Internet already not remember as

**Comment by asimkhan0** [comment]
@Sergey-Ssn Could you please share the solution?

**Comment by Sergey-Ssn** [comment]
> @Sergey-Ssn Could you please share the solution?

I ordered a freelance job
